ivadomed/config/config_bids.json

```diff
@@ -99,7 +99,7 @@
         },
         {
             "name": "datatype",
-            "pattern": "[/\\\\]+(func|anat|fmap|dwi|meg|eeg|microscopy|ct)[/\\\\]+"
```

Comment:
Just a minor question about this, I guess we are never going to be dealing with fMRI, field map then, is that the case? EEG I can see that it will be a LONG while until we support it given the bleeding edge stuff @mpompolas has been doing...?